spec/lib/tasks/migrate_lockbox_action_text_rake_task_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe "lockbox:migrate_action_text" do
+  include_context "rake"
+
+  let(:task_path) { "lib/tasks/migrate_lockbox_action_text" }
+
+  # rubocop:disable RSpec/NamedSubject
+  it "calls Lockbox.migrate with ActionText::RichText" do
+    expect(Lockbox).to receive(:migrate).with(ActionText::RichText)
+    subject.invoke
+  end
```

Comment:
The current assertion is only about the call happening. Could we test that `subject.invoke` causes a message with `body` to populate its `body_cyphertext` and that `migrated_body` matches the original `body`? (If I understand that `migrated_body` is the plain attribute matching the encrypted DB field?)